controller/xt_hs_smartrouting/impl.go

```diff
@@ -0,0 +1,68 @@
+package xt_hs_weighted
+
+import (
+	"github.com/netfoundry/ziti-fabric/controller/xt"
+	"math"
+)
+
+func NewFactory() xt.Factory {
```

Comment:
Wonder if it might be worth writing a short descriptive comment for each of these implementations, describing what they do?

Reply:
Will do